Examples/Swift/ExtrusionsExample.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExtrusionsExample.swift
+//  Examples
+//
+//  Created by Jordan Kiley on 5/10/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+import Mapbox
+
+@objc(ExtrusionsExample_Swift)
+
+class ExtrusionsExample: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.lightStyleURL(withVersion: 9))
+        
+        // Center the map on the Colosseum in Rome, Italy.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 41.8902, longitude: 12.4922), animated: false)
+
+        // Set the map view camera's pitch and distance.
+        mapView.camera = MGLMapCamera(lookingAtCenter: mapView.centerCoordinate, fromDistance: 600, pitch: 60, heading: 0)
+        mapView.delegate = self
+        
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Access the Mapbox Streets source and use it to create a `MGLFillExtrusionLayer`.
+        let source = style.source(withIdentifier: "composite")
+        let layer = MGLFillExtrusionStyleLayer(identifier: "buildings", source: source!)
+        layer.sourceLayerIdentifier = "building"
+        
+        // Filter out buildings that should not extrude.
+        layer.predicate = NSPredicate(format: "extrude != false AND height >=0")
```

Comment:
According to the [Mapbox Streets source documentation](https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#building):

> The `extrude` field is `true` or `false` depending one whether the object should be included in 3D-extrusion renderings. For example a complex building might have various `building:part` objects mapped with different heights, in addition to a building object representing the footprint of the entire building. Only the `building:part` objects are needed for 3D rendering, so the full footprint outline will have an `extrude` value of `false`.

For a lower-level understanding of this value, see the [OpenStreetMap documentation](https://wiki.openstreetmap.org/wiki/Simple_3D_buildings#Building_outline) about 3D building outlines.